pyas2/models.py

```diff
@@ -39,6 +43,36 @@ class PublicCertificate(models.Model):
         verbose_name=_('Verify Certificate'), default=True,
         help_text=_('Uncheck this option to disable certificate verification.'))
 
+    certificate_valid_from = models.DateTimeField(null=True, blank=True)
```

Comment:
rename the fields as valid_from and valid_to

Reply:
done.